private/buf/bufgen/bufgen.go

```diff
@@ -40,6 +48,77 @@ type ExternalConfigVersion struct {
 	Version string `json:"version,omitempty" yaml:"version,omitempty"`
 }
 
+// Generator generates Protobuf stubs based on configurations.
```

Comment:
The `Generator` stuff is moved from `bufgenv1.go` (previously known as `bufgen.go`)